expat/Makefile.am

```diff
@@ -28,6 +28,9 @@
 # OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE
 # USE OR OTHER DEALINGS IN THE SOFTWARE.
 
+#page
```

Comment:
I noticed `#page` at a few places. Which tool are these for?

Reply:
It is a custom configuration I use for GNU Emacs.  I have changed it to the more standard `^L`, but I can remove it.